imagehash.go

```diff
@@ -60,11 +61,52 @@ func (h *ImageHash) GetHash() uint64 {
 }
 
 // GetKind method returns a kind of image hash.
-func (h *ImageHash) GetKind() hashKind {
+func (h *ImageHash) GetKind() Kind {
 	return h.kind
 }
 
 // Set method sets a bit of index.
 func (h *ImageHash) Set(idx int) {
 	h.hash |= 1 << uint(idx)
 }
+
+const strFmt = "%v:%0x"
+
+// FromString returns an image hash from a hex representation
+func FromString(s string) (*ImageHash, error) {
```

Comment:
Can you add unit tests to this function?
Two things will be needed.

1. Valid string case. (string value should be fit for uint64)
2. Invalid string case. (string value which is not fit for unit64)